components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,283 @@
+                </div>
+              ))}
+              <div className={styles["row"]} />
+              {invites.length != 5 && (
+                <IconButton
+                  variant="contained"
+                  onClick={addRow}
+                  // style={{ backgroundColor: 'transparent' }}
+                  className={styles["add"]}
+                >
+                  <Icon type={"inviteAdd"} className={styles["addIcon"]} />
+                  New Invitation
+                </IconButton>
+              )}
+            </div>
+          </div>
+          <div className={styles["actions"]}>
+            <DialogActions>
+              <ThemeProvider theme={theme}>
+                <Button
+                  style={{ textTransform: "none" }}
+                  className={styles["buttons"]}
+                  variant="outlined"
+                  onClick={handleClose}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  type="submit"
+                  className={styles["buttons"]}
+                  variant="contained"
+                >
+                  Send Invites
+                </Button>
+              </ThemeProvider>
+            </DialogActions>
+          </div>
+        </form>
+      </div>
+    );
+  };
+
+  const renderSuccessPage = () => {
+    return (
+      <div className={styles["success"]}>
+        <div className={styles["content"]}>
+          <h1 className={styles["header"]}>Success!</h1>
+          <p className={styles["message"]}>
+            All emails have been sent to their respective addresses
+          </p>
+          <div className={styles["actions"]}>
+            <ThemeProvider theme={theme}>
+              <Button
+                style={{ textTransform: "none" }}
+                className={styles["buttons"]}
+                variant="outlined"
+                onClick={handleClose}
+              >
+                Close
+              </Button>
+              <Button
+                type="submit"
+                className={styles["invite-more"]}
+                variant="contained"
+                onClick={() => setSent(false)}
+              >
+                Invite More
+              </Button>
+            </ThemeProvider>
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <div>
+      <ThemeProvider theme={theme}>
+        <Button
+          variant="contained"
+          style={{ textTransform: "none" }}
+          onClick={handleClickOpen}
+        >
+          Invite Admin
+        </Button>
+
+        <Dialog
+          classes={{
+            paper:
+              count > 3
+                ? styles["biggestModal"]
+                : count < 3
+                ? styles["modal"]
+                : styles["biggerModal"],
+          }}
+          maxWidth="md"
+          fullWidth={true}
+          open={open}
+          onClose={handleClose}
```

Comment:
There is a flickering issue when we close the modal, I think you can fix this by updating the variables before opening the modal (similar to how me and Jacob fixed the add transactions popover)